docs/source/data/resolvers.mdx

```diff
@@ -461,6 +461,14 @@ type Author {
 
 If the resolver for the `books` field returns an array of objects that each contain a `title` field, then you can use a default resolver for the `title` field. The default resolver will correctly return `parent.title`.
 
+## Resolving types
+
+There are GraphQL types that enable you to define a field that returns one of multiple possible object types (i.e. [unions and interfaces](../schema/unions-interfaces/#resolving-a-union)). To resolve a field that can return different object types, you must define a `__resolveType` function to inform Apollo Server _which_ object type to return.
+
+> [See examples of the `__resolveType` function](../schema/unions-interfaces/#resolving-a-union).
+
+If you using Apollo Federation and looking for more information on resolving entity types, see [Resolving Entities](https://www.apollographql.com/docs/federation/entities/#resolving-entities).
```

Comment:
you -> you're

With the header change above, maybe make _another_ header called like "Resolving federated entities" and just have the body be a sentence like "See [Resolving entities](https://www.apollographql.com/docs/federation/entities/#resolving-entities)"